src/ui/hooks/recordings.ts

```diff
@@ -105,7 +107,16 @@ const GET_MY_RECORDINGS = gql`
 
 export function useGetRecordingId() {
   const { id } = useRouter().query;
-  return Array.isArray(id) ? id[0] : id!;
+  return extractIdAndSlug(id).id!;
+}
+
+export function getRecordingURL(recording: Recording) {
+  let slug = "";
+  if (recording.title) {
+    slug = `${slugify(recording.title).toLowerCase()}/`;
```

Comment:
This is super pedantic but I think the `/` should on line 119 and this should be 

```
    slug = slugify(recording.title).toLowerCase()
```

Reply:
That was my first impl but we can't guarantee a title (e.g. from node recordings) so the logic below gets messy if we have to conditionally add the slash below based on `slug` being valued. It's doable but not my preference.

Reply:
What about `["recording",slug,recording.id].filter(Boolean).join("/")`? Maybe I'm just making things worse 😆

Reply:
or `compact(["recording", slug, recording.id].join("/")` is probably clearer